packages/table/src/common/_loading.scss

```diff
@@ -14,25 +14,29 @@
   }
 }
 
-.bp-table-cell,
-.bp-table-header {
-  &.pt-loading {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    cursor: default;
+@mixin hide-text() {
+  // Always set color to transparent so cell content can fade-in when transitioning from the
+  // loading state
+  // stylelint-disable-next-line declaration-no-important
+  color: transparent !important;
+}
 
-    // Always set color to transparent so cell content can fade-in when transitioning from the
-    // loading state
-    // stylelint-disable-next-line declaration-no-important
-    color: transparent !important;
-    pointer-events: none;
-  }
+@mixin disable-interaction() {
+  cursor: default;
+  pointer-events: none;
+}
+
+@mixin vertical-center() {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
 
+@mixin table-skeleton($skeleton-height) {
```

Comment:
because headers accept arbitrary children, it's possible that users will use the `.pt-skeleton` class on children. is it okay to be applying additional styles to this class under their noses? if so, we could stick these additional properties on `.bp-table-header .pt-skeleton`